Resources/template.json

```diff
@@ -446,6 +458,14 @@
                 "name": "CourseServiceSettings__ApiKey",
                 "value": "[parameters('courseApiKey')]"
               },
+              {
+                "name": "ApprenticeshipServiceSettings__ApiUr",
```

Comment:
Should the name of this be ApprenticeshipServiceSettings__ApiUr or ApprenticeshipServiceSettings__ApiUrl (with an l on the end)?